pkg/consts/consts.go

```diff
@@ -25,3 +25,4 @@ const (
 )
 
 const LatestKanisterToolsImage = "ghcr.io/kanisterio/kanister-tools:v9.99.9-dev"
+const KanisterToolsImage = "ghcr.io/kanisterio/kanister-tools:0.88.0"
```

Comment:
We have a constant in `pkg/function/utils.go`. Please search and replace its uses with this constant in a separate PR
```suggestion
const KanisterToolsImage = "ghcr.io/kanisterio/kanister-tools:0.89.0"
```